paf-mvp-core-js/src/express/node.ts

```diff
@@ -139,9 +139,9 @@ export class Node implements INode {
    * Begin handling of a request.
    * Should be called first.
    */
-  beginHandling = (req: Request, res: Response, next: NextFunction) => {
+  beginHandling = (req: any, res: Response, next: NextFunction) => {
     const { endPointName } = this.getRequestConfig(req);
-    this.logger.Info(endPointName);
+    this.logger.Info(`${endPointName} @${req.correlationId()} - START`);
```

Comment:
As you explained to me offline, the call to `req.correlationId()` will get the header or create a new id if it doesn't exist.
- why don't you use this everywhere, rather than doing `req.header('onekey-correlation-id')`? It would seem more safe to use `req.correlationId()` everywhere
- if not, then I think this call deserves a comment, otherwise it is confusing why you use a different method here than elsewhere

Reply:
`correlationId()` is used in the first middleware of each endpoint to make sure a correlation-id is generated if the received request doesn't have one.
For subsequent middlewares, we can get the value directly from the header as we are sure it has already been set. This will avoid the use of `any` type.